### DISCUSSION TITLE
why bg color dropdown property doesnt work?

### DISCUSSION THREAD

**Discussion by andyargg** [flet-dev/flet#2884] [Q&A]
### Question

      import flet as ft
      
      def main(page: ft.Page):
          def button_clicked(e):
            
              page.update()
    
     
        dd = ft.Dropdown(
            width=300,
            bgcolor=ft.colors.RED,
            options=[
                ft.dropdown.Option("Red"),
                ft.dropdown.Option("Green"),
                ft.dropdown.Option("Blue"),
            ],
        )
        page.add(dd, )

ft.app(target=main)

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Subscribe to this issue please: #2814
